test/unit/source/deinitialization.d

```diff
@@ -0,0 +1,150 @@
+// global.deinitialize
+unittest
+{
+    import dmd.globals : global;
+
+    static void assertStructsEqual(T)(const ref T a, const ref T b) @nogc pure nothrow
+        if (is(T == struct))
```

Comment:
As this is new code:

> Constraints on declarations should have the same indentation level as their declaration:

https://dlang.org/dstyle.html#phobos_declarations

Reply:
Ok.